doc/api/crypto.md

```diff
@@ -522,6 +522,8 @@ is invalid according to [NIST SP 800-38D][] or does not match the value of the
 
 The `decipher.setAuthTag()` method must be called before
```

Comment:
I would just modify this sentence instead to clarify, something like:

```
The `decipher.setAuthTag()` method must be called before
[`decipher.update()`][] for `CCM` mode or before [`decipher.final()`][] for
`GCM` and `OCB` modes. `decipher.setAuthTag()` can only be called once.
```

Reply:
Great, thanks!  I just pushed a correction with this advice.